rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -1215,7 +1222,7 @@ def _load_model_class(
             data_signature=model_data_example.get_signature(),
             label_data=label_data,
             entity_tag_specs=entity_tag_specs,
-            config=copy.deepcopy(meta),
+            config=copy.deepcopy(config),
```

Comment:
why the `deepcopy`?

Reply:
Because it was always like that before 😛

Reply:
lol, should we drop it?

Reply:
I had a slight worry that it was done because the model manipulated the config (i.e. like `DIETClassifier` does). And didn't want some strange hard to find bug to pop up later before of this change.... but we could YOLO and change it?

Reply:
Seems DIET just uses the load inherited from RasaModel atm which just does: https://github.com/RasaHQ/rasa/blob/08bf76bd70cc58784556e69c60b03570149f1119/rasa/utils/tensorflow/models.py#L422
And DIET2BOW and DIET2DIET don't change anything either (but do the copy in model_load call as well). So .... :D  

(In general, I'd do a copy in all __init__s etc. but I think that should go into that refactor configs task one day :D)

Reply:
The config actually ends up going all the way into `tf.keras.models.Model` via kwargs - although I have no idea if it does anything with them (probably not).